ansible/roles-infra/infra-openshift-cnv-resources/tasks/stop_instances.yaml

```diff
@@ -1,18 +1,19 @@
 ---
 - name: Get a list of VMIs
   kubernetes.core.k8s_info:
-    kind: VirtualMachineInstance
+    kind: VirtualMachine
     namespace: "{{ openshift_cnv_namespace }}"
-  register: r_vmi_list
-  until: r_vmi_list is success
+  register: r_vm_list
+  until: r_vm_list is success
   retries: "{{ openshift_cnv_retries }}"
   delay: "{{ openshift_cnv_delay }}"
 
 
-- name: Stop the Virtual Machines
-  loop: "{{ r_vmi_list.resources }}"
+- name: Stop the Virtual Machines (running)
+  loop: "{{ r_vm_list.resources }}"
   loop_control:
     loop_var: _instance
+  when: _instance.spec.running|default(False) == true
```

Comment:
I'd like us to be consistent with booleans being false / true. Also Ansible-lint usually suggest a space around the |. Finally I'd write this as ... | default(false) | bool instead of comparing to a bool value.